hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java

```diff
@@ -70,8 +68,8 @@ static TransactionHandlers provideTransactionHandlers(
                 fileHandlers.fileAppendHandler(),
                 fileHandlers.fileSystemDeleteHandler(),
                 fileHandlers.fileSystemUndeleteHandler(),
-                adminHandlers.freezeHandler(),
-                networkHandlers.networkUncheckedSubmitHandler(),
+                networkAdminHandlers.freezeHandler(),
+                networkAdminHandlers.networkUncheckedSubmitHandler(),
```

Comment:
Are we keeping unchecked submit? I seem to remember it was going away, but am not 100% sure on that

Reply:
I'm checking with Lev (owner of Network service). If he doesn't know then we'll need to discuss it before this method is implemented (it's just a stub now). Either way, I don't think it should hold up this PR.

Reply:
Lev says he doesn't think this one is going away.